enclave-runner/src/usercalls/mod.rs

```diff
@@ -714,9 +717,12 @@ impl EnclaveState {
                                     println!("Attaching debugger");
                                     #[cfg(all(unix, not(target_abi = "musl")))]
                                     trap_attached_debugger(usercall.tcs_address() as _).await;
-                                    Err(EnclaveAbort::Exit {
-                                        panic: EnclavePanic::from(buf.into_inner()),
-                                    })
+                                    let panic = EnclavePanic::from(buf.into_inner());
+                                    if enclave_clone.forward_panics {
+                                        panic!("{}", &panic);
```

Comment:
I think just using eprintln! is better as this one produces messages like the following:
```
thread 'main' panicked at 'Enclave panicked: thread '<unnamed>' panicked at 'spawned thread panicking...', src/main.rs:6:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
', enclave-runner/src/usercalls/mod.rs:722:41
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
@jethrogb Did you see this?

Reply:
I'm indifferent about printing but I do like panicking instead of plain exiting.